ql/src/java/org/apache/hadoop/hive/ql/ServiceContext.java

```diff
@@ -30,6 +30,7 @@ public class ServiceContext {
   private final String clusterId;
   private final Supplier<String> host;
   private final Supplier<Integer> port;
+  public static final String QUERY_ID = "Query ID";
```

Comment:
I am not sure if this is the correct place to keep this variable, The Javadoc of ``ServiceContext`` itself says ``Information context about the service, most probably HiveServer2.``